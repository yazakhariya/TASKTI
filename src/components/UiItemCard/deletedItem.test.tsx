import { render, screen } from '@testing-library/react'
import { describe, expect, it } from 'vitest'
import DeletedItem from './deletedItem'

describe('deleted card item', () => {
  it('should render properly', () => {
    render(<DeletedItem img={''} name={''} price={''} />)
    expect(screen.getByTestId('my-test-id')).toBeDefined()
  })
})

