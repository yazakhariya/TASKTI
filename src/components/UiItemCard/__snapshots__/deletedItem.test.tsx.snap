// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`deleted card item > should match snapshot 1`] = `
<div
  class="_wrapper_8fd2df _item_8fd2df"
  data-testid="my-test-id"
>
  <img
    alt="Изображение товара "
    class="_productImg_8fd2df _deleted_8fd2df"
    src=""
  />
  <div
    class="_cartContentBox_8fd2df _deletedItem_8fd2df"
  >
    <div
      class="_textBox_8fd2df _deleted_8fd2df"
    >
      <span
        class="_itemName_8fd2df"
      />
      <span
        class="_itemPrice_8fd2df"
      />
    </div>
    <button
      class="_button_8fd2df _deletedButtonState_8fd2df"
      type="button"
    >
      <img
        alt="Кнопка добавления продукта в корзину"
        src="/@fs/C:/Users/%D0%AF%D0%BD%D0%B0/Desktop/t1/TASKTI/public/addtocart.svg"
      />
    </button>
  </div>
</div>
`;

exports[`deleted card item > should render properly 1`] = `
<div
  class="_wrapper_8fd2df _item_8fd2df"
  data-testid="my-test-id"
>
  <img
    alt="Изображение товара "
    class="_productImg_8fd2df _deleted_8fd2df"
    src=""
  />
  <div
    class="_cartContentBox_8fd2df _deletedItem_8fd2df"
  >
    <div
      class="_textBox_8fd2df _deleted_8fd2df"
    >
      <span
        class="_itemName_8fd2df"
      />
      <span
        class="_itemPrice_8fd2df"
      />
    </div>
    <button
      class="_button_8fd2df _deletedButtonState_8fd2df"
      type="button"
    >
      <img
        alt="Кнопка добавления продукта в корзину"
        src="/@fs/C:/Users/%D0%AF%D0%BD%D0%B0/Desktop/t1/TASKTI/public/addtocart.svg"
      />
    </button>
  </div>
</div>
`;
